web/src/components/organisms/entity/EntityTable.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react'
+import { Table, TableProps, TableHead, TableHeadProps, TableBody, TableBodyProps } from '../../atoms';
+import { EntityTableHeader, EntityTableHeaderProps, EntityTableRow, EntityTableRowProps } from '../../molecules';
+import { Entity } from '../../../types';
+
+export interface EntityTableProps {
+    tableProps?: TableProps,
+    headProps?: TableHeadProps, 
+    headerProps?: EntityTableHeaderProps,
+    bodyProps?: TableBodyProps,
+    rowProps?: EntityTableRowProps,
+    entities: Entity[],
+};
+
+export type EntityTable = React.FunctionComponent<EntityTableProps>;
+
+export const EntityTable: EntityTable = ({ tableProps, headProps, headerProps, bodyProps, rowProps, entities }: { tableProps: TableProps, headProps: TableHeadProps, headerProps: EntityTableHeaderProps, bodyProps: TableBodyProps, rowProps: EntityTableRowProps, entities: Entity[] }) => {
+    const mapEntity = (entity: Entity) => <EntityTableRow {...{...rowProps, entity}}></EntityTableRow>;
+    
+    const EntityTableRows = React.useCallback(() => (
+        <React.Fragment>{entities.map(mapEntity)}</React.Fragment>
+    ), [rowProps, entities]);
+    
+    return (
+        <Table {...tableProps}>
+            <TableHead {...headProps}>
```

Comment:
I think neater to have the `TableHead` inside the `EntityTableHeader` as the latter only makes sense in the context of a table head and it tidies up this component which is getting unwieldy.
In which case the complimentary `EntityTableBody` would make sense - a `TableBody` which includes the `EntityTableRows`

Reply:
That's actually initially how I had it, but I ended up changing it because it broke consistency with the general table API (e.g. `TableBody` should be a context, not include rows). 

As these are `organisms`, they could be considered at a higher level of abtraction but I think they need a name that makes it clearer that they are compositions of  `Head`+`Header` and `Body`+`Row[]`. Can go with `EntityTableHeader = TableHead + EntityTableRow` as that's unambiguous, but need an equivalent for `EntityTableBody` (something like `EntityTableData", but I don't like that very much!) Any ideas?